### PR TITLE
Add a Recommended Permissions FAQ section in README

### PR DESCRIPTION
**Pull request by Scrub1492**
Adds a section about recommended permissions for Firefox in README.md. 

This is a step towards more intuitive UI to grant permissions for Firefox, #127 .
Related: #330 , https://github.com/themoeway/yomitan/issues/330#issuecomment-1865891179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,6 +350,11 @@ A setup guide can be found [here](https://github.com/themoeway/yomitan-mecab-ins
 
 ## Frequently Asked Questions
 
+**I can't scan text in Firefox!**
+
+In Firefox's Manifest V3, host permissions are treated as opt-in. For Yomitan to work properly, the recommended permissions
+must be explicitly set. Go to the `Recommended Permissions (Important)`section and check `Enable recommended permissions`.
```
Comment (djahandarie) [inline README.md:356]: section of what?

Comment (toasted-nutbread) [inline README.md:356]: space between
```
`section
```

Comment (Scrub1492) [reply README.md]: added 'In the Yomitan welcome page'.

Comment (Scrub1492) [reply README.md]: fixed.

**Review by djahandarie — APPROVED** [review]
Thanks! I think if we fix the UI to be more prominent we can eventually get rid of this, but it should help a bit in the meantime.

**Pull request merged**